recipe/meta.yaml

```
@@ -88,6 +86,7 @@ test:
     - test -f $PREFIX/lib/libgirepository-1.0${SHLIB_EXT}  # [not win]
     - if not exist %PREFIX%\Library\bin\girepository-1.0-1.dll exit 1  # [win]
     - if not exist %PREFIX%\Library\lib\girepository-1.0.lib exit 1  # [win]
+
 about:
   home: https://wiki.gnome.org/action/show/Projects/GObjectIntrospection
```

```suggestion
  home: https://gi.readthedocs.io/
```

Looks like the home url changed to this: https://wiki.gnome.org/Projects/GObjectIntrospection but that now points to the readthedocs.